README.md

```diff
@@ -205,3 +205,25 @@ This is our basic integration flow:
 4. Merge to master.
 
 We keep stable branches receiving bug and security fixes for long term releases.
+
+### Releasing a new version
```

Comment:
Should this be `##` ? So it's the same as `Integration flow`.
Also, have you considered creating a separated file for this and link it here?

Reply:
yes, I have considered both - the integration flow is what we have now that most closely resembles a "developer" section, and for creating a new file we still don't have enough content, but certainly looking forward to adding it once we get more material.

Reply:
I prefer the option of hiding content that it's only useful for a few people and leave a simpler README for everyone, but this is just a matter of preference.

Reply:
yes, and I expect we will do that once we add a bit more content.